src/Inventory.java

```diff
@@ -0,0 +1,275 @@
+        this.inventory.put(CURRENT_ITEM_ID, item);
+        CURRENT_ITEM_ID++;
+
+        System.out.format("\n%d <%s> @ %.2f has been added to inventory.\n", item.getQuantity(), item.getName(), item.getPrice());
+
+        whatYouWant();
+    }
+
+    private void deleteItem(){
+
+        int id =0;
+        while (id == 0) {
+            System.out.println("\nWhat is the ID of the item you would like to delete?");
+            id = getValidInt(this.input.nextLine());
+            if (this.inventory.get(id) == null){
+                System.out.println("\nNo such item exists at that ID.\n");
+                id = 0;
+            }
+        }
+
+        this.inventory.remove(id);
+        whatYouWant();
+    }
+
+    private void sellItem(){
+        Item item = getItem();
+        int itemQty = item.getQuantity();
+        System.out.println("\nWhat is the quantity to sell?");
+        int qty = getValidQty(this.input.nextLine());
+
+        while (qty > itemQty) {
+            System.out.println("\nThere is not enough in stock!");
+            System.out.println("\nWhat is the quantity to sell?");
+            qty = getValidQty(this.input.nextLine());
+        }
+
+        item.setQuantity(itemQty-qty);
+        System.out.format("\n%d <%s> have been removed from inventory.", qty, item.getName());
+
+        whatYouWant();
+    }
+
+    private void itemOnSale(){
+        Item item = this.getItem();
+        System.out.println("\nWhat is the sale start date? <dd/mm/yyyy>");
+        item.setSaleStart(getValidDate());
+        System.out.println("\nWhat is the sale end date? <dd/mm/yyyy>");
+        item.setSaleEnd(getValidDate());
+        System.out.println("\nWhat is the sale price?");
+        item.setSalePrice(getValidPrice(this.input.nextLine()));
+
+        whatYouWant();
+    }
+
+    private void print(int id){
+        String str = String.format("ID#%d %s\n", id, inventory.get(id).toString());
+
+        System.out.format(str);
+        for (int i=0; i<str.length(); i++){
+            System.out.print('-');
+        }
+        System.out.print("\n");
+    }
+
+    private void printAll(){
+        if (inventory.isEmpty()){
+            System.out.println("\nThere are no items in the inventory.");
+        }
+        System.out.println("");
+        for(Integer id : inventory.keySet()){
+            print(id);
+        }
+        whatYouWant();
+    }
+
+    private void updatePrice(){
+        Item item = getItem();
+        System.out.println("\nWhat is the new price?");
+        double price = getValidPrice(this.input.nextLine());
+        item.setPrice(price);
+        whatYouWant();
+    }
+
+    private void updateQuantity(){
+        Item item = getItem();
+        System.out.println("\nWhat is the new quantity?");
+        int qty = getValidQty(this.input.nextLine());
+        item.setQuantity(qty);
+        whatYouWant();
+    }
+
+    private void exit(){
+        System.exit(0);
```

Comment:
In general you should try to avoid a `System.exit()`.  This should also be the responsibility of the `main()` method as `Inventory` shouldn't be allowed to execute the entire program.